### PR TITLE
add quarkus and micronaut

### PR DESCRIPTION
**Pull request by jamesward**
Replaces #74

### CODE REVIEW

**Comment by bshaffer** [comment]
We need to add these directly to `.github/workflows/tests.yml`

**Comment by jamesward** [comment]
Added. Thanks for the tip!

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -98,6 +98,18 @@ jobs:
...
         wrapper-directory: run/ktor-hello-world
 
+    - name: Cloud Run (Micronaut) - Gradle
+      uses: eskatos/gradle-command-action@v1
+      with:
+        arguments: test --console=plain --info
+        build-root-directory: run/micronaut-hello-world
+        wrapper-directory: run/micronaut-hello-world
+
+    - name: Cloud Run (Quarkus) - Gradle
```
Comment (bshaffer) [inline .github/workflows/tests.yml:108]: nit: name should be `Cloud Run (Quarkus) - Maven`

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -98,6 +98,18 @@ jobs:
...
+      run: |
+        cd ${GITHUB_WORKSPACE}/run/quarkus-hello-world
+        ./mvnw test
```
Comment (bshaffer) [inline .github/workflows/tests.yml:111]: Another nit: let's use `./mvnw test -q`, otherwise we get 5K+ lines of output instead of ~100

**Comment by jamesward** [comment]
All is passing now :)

**Review by bshaffer — APPROVED** [review]
I ran them locally and deployed to cloud run, and also ran with buildpack/Docker. Everything worked beautifully, nice job!

**Pull request merged**